samples/openTelemetryTracing.js

```diff
@@ -32,103 +32,107 @@
 
 const SUBSCRIBER_TIMEOUT = 10;
 
+// [START opentelemetry_tracing]
```

Comment:
Yeah, I think the person who originally made this just came up with one? I think maybe it deserves several samples, e.g. one for Cloud Trace, another for console output, etc.

Reply:
I think we shouldn't do console output and just focus on Cloud Trace as the default sample (since that's also what our open telemetry team's documentation points to).

Reply:
I do think that's the primary thing we want to help people with, 'cause it's probably what they want to do. But I do think we ought to at least e.g. have a Medium article talking about how to output to console, Grafana, etc. Those will be useful to people too.